package.json

```diff
@@ -37,6 +37,7 @@
     "commander": "^13.1.0",
     "eslint": "^9.23.0",
     "execa": "^8.0.1",
```

Comment:
Can you remove `execa` from the dependencies?

Reply:
because we still use `execa` in `dependencies.test.ts`.
The logic of the standard input/output tests is not rewritten in this PR.

Reply:
I wasn't sure how to rewrite the test logic, so I left it as is

Reply:
I understood! Thanks.